src/elements/content-sidebar/SidebarNavSignButton.scss

```diff
@@ -14,14 +14,12 @@
     margin-top: 2px; //for visual alignment
 }
 
-.is-responsive-web {
-    @include breakpoint($medium-screen) {
```

Comment:
no issues with removing the `is-responsive-web` class since the specificity for the base styles are already low